app/templates/components/public/call-for-speakers.hbs

```diff
@@ -1,32 +1,27 @@
 <h1 class="ui header">
   {{t 'Call for Speakers'}}
-  {{#if callForSpeakers.isClosed}}
-    <a class="ui basic red label">{{t 'Closed'}}</a>
-  {{else}}
-    <a class="ui basic green label">{{t 'Open'}} </a>
+  {{#if speakersCall.isOpen}}
+    <a class="ui basic green label">{{t 'Open'}}</a>
     <div class="sub header">
-      {{t 'Call for Speakers Open until'}} Fri, Oct 13 2017 5:00 pm
+      {{t 'Call for Speakers Open until'}} {{moment-format speakersCall.endsAt 'ddd, MMM DD HH:mm A'}}
     </div>
+  {{else}}
+    {{#if speakersCall.isInFuture}}
```

Comment:
@harshitagupta30 This is causing a lot of code duplication can we have it as  :) 
```handlebars 
<a class="ui basic red label">{{t 'Closed'}}</a>
<div class="sub header">
  {{#if speakersCall.isInFuture}}
     {{t 'Call for Speakers will open at'}} {{moment-format speakersCall.startsAt 'ddd, MMM DD HH:mm A'}}
  {{else}}
    {{t 'Call for Speakers closed at'}} {{moment-format speakersCall.endsAt 'ddd, MMM DD HH:mm A'}}
  {{/if}}
</div>
```